docker/encfs/Dockerfile.encfs

```diff
@@ -1,8 +1,19 @@
+FROM mcr.microsoft.com/devcontainers/go:dev-1.22-bookworm AS build

```

Comment:
Just wondering, when do we want to use a dev container image? 
Currently, I think the repo is using two types of go images (both with 1.22.1 available)
> mcr.microsoft.com/oss/go/microsoft/
> mcr.microsoft.com/devcontainers/go

Reply:
The dev image has make (amongst other things) in it, so we're using it for the build of SKR and ENCFS. As for the oss/go/microsoft image, we're using that in the attestation container, which I assume is because of the crypto libraries in the fips image being used. However, Dominic and Takuro are the ones who created that container (I think) and so they would have a better idea of why that image is used instead of the devcontainer. I just didn't see a need to update that image.